es-models/gene_centric/gene_centric.mapping.yaml

```diff
@@ -196,6 +196,8 @@ properties:
                         type: keyword
                       dbsnp_rs:
                         type: keyword
+                      dbsnp_val_status:
```

Comment:
Updates to things not part of the gdc_from_graph should probably be in a separate pr.